Can't publish the initial version as a prerelease

```
❯ np --version
2.16.0

❯ np

Publish a new version of daily-version (0.0.0)

? Select semver increment or specify new version premajor 	1.0.0-0

✖ Command failed: npm view --json daily-version dist-tags
npm ERR! code E404
npm ERR! 404 Registry returned 404 for GET on https://registry.npmjs.org/daily-version
npm ERR! 404 
npm ERR! 404  'daily-version' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/bfred/.npm/_logs/2017-08-29T08_00_27_381Z-debug.log
{
  "error": {
    "code": "E404",
    "summary": "Registry returned 404 for GET on https://registry.npmjs.org/daily-version",
    "detail": "\n 'daily-version' is not in the npm registry.\nYou should bug the author to publish it (or use the name yourself!)\n\nNote that you can also install from a\ntarball, folder, http url, or git url."
  }
}
```

Instead of using a premajor, I ended up publishing as `0.9.0`

--- a/source/npm-util.js
+++ b/source/npm-util.js
@@ -50,7 +50,7 @@ exports.prereleaseTags = async packageName => {
 		tags = Object.keys(JSON.parse(stdout))
 			.filter(tag => tag !== 'latest');
 	} catch (error) {
-		if (/is not in the npm registry\./.test(error.details) {
+		if (/is not in the npm registry\./.test(JSON.parse(error.stdout).detail)) {
 			tags = [];
 		} else {
 			throw error;

--- a/source/npm-util.js
+++ b/source/npm-util.js
@@ -50,7 +50,7 @@ exports.prereleaseTags = async packageName => {
 		tags = Object.keys(JSON.parse(stdout))
 			.filter(tag => tag !== 'latest');
 	} catch (error) {
-		if (/is not in the npm registry\./.test(JSON.parse(error.stdout).detail)) {
+		if (JSON.parse(error.stdout).code === 'E404') {
 			tags = [];
 		} else {
 			throw error;

--- a/source/npm-util.js
+++ b/source/npm-util.js
@@ -44,15 +44,13 @@ exports.collaborators = async packageName => {
 exports.prereleaseTags = async packageName => {
 	ow(packageName, ow.string);
 
-	let tags;
+	let tags = [];
 	try {
 		const {stdout} = await execa('npm', ['view', '--json', packageName, 'dist-tags']);
 		tags = Object.keys(JSON.parse(stdout))
 			.filter(tag => tag !== 'latest');
 	} catch (error) {
-		if (JSON.parse(error.stdout).code === 'E404') {
-			tags = [];
-		} else {
+		if (JSON.parse(error.stdout).code !== 'E404') {
 			throw error;
 		}
 	}

--- a/source/npm-util.js
+++ b/source/npm-util.js
@@ -46,7 +46,7 @@ exports.prereleaseTags = async packageName => {
 
 	let tags;
 	try {
-		const stdout = await execa.stdout('npm', ['view', '--json', packageName, 'dist-tags']);
+		const {stdout} = await execa('npm', ['view', '--json', packageName, 'dist-tags']);
 		tags = Object.keys(JSON.parse(stdout))
 			.filter(tag => tag !== 'latest');
 	} catch (error) {

--- a/source/npm-util.js
+++ b/source/npm-util.js
@@ -50,7 +50,7 @@ exports.prereleaseTags = async packageName => {
 		tags = Object.keys(JSON.parse(stdout))
 			.filter(tag => tag !== 'latest');
 	} catch (error) {
-		if (error.code === 'E404') {
+		if (/is not in the npm registry\./.test(error.details) {
 			tags = [];
 		} else {
 			throw error;

